integration-tests/test_debug_remaining_issues.py

```
@@ -0,0 +1,213 @@
+            print(f"📊 Response Status: {resp.status}")
+            
+            if resp.status == 500:
+                error_text = await resp.text()
+                print(f"❌ 500 Error Details:")
+                print(f"   Error: {error_text}")
+                
+                # Try to parse JSON error
+                try:
+                    error_json = json.loads(error_text)
+                    print(f"   Parsed Error: {json.dumps(error_json, indent=2)}")
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)